src/main/flight/position.h

```diff
@@ -40,3 +40,4 @@ int32_t getEstimatedAltitudeCm(void);
 float getAltitude(void);
 float getAltitudeAsl(void);
 int16_t getEstimatedVario(void);
+bool isAltitudeAvailable(void);
```

Comment:
No return on EOF
```suggestion
bool isAltitudeAvailable(void);

```

Reply:
thanks, fixed, I'll try not to forget them